src/Controllers/CopyPatrol.php

```diff
@@ -108,6 +108,10 @@ protected function handleGet() {
 				$records[$key]['editor_page_dead'] = false;
 				$records[$key]['editor_talk_dead'] = false;
 			}
+			if ( $record['status_user'] ) {
+				$records[$key]['reviewed_by_url'] = $this->getUserPage( $record['status_user'] );
+			}
+			$records[$key]['review_timestamp'] = $this->formatTimestamp( $record['review_timestamp'] );
```

Comment:
Nope, we aren't letting unauthenticated users perform reviews anymore, per the latest code in master. You can get rid of it now.